pkg/skaffold/runner/build_deploy.go

```diff
@@ -105,12 +105,8 @@ func (r *SkaffoldRunner) DeployAndLog(ctx context.Context, out io.Writer, artifa
 		return err
 	}
 
-	if r.runCtx.Opts.PortForward.Enabled {
-		if err := forwarderManager.Start(ctx); err != nil {
-			logrus.Warnln("Error starting port forwarding:", err)
-		} else {
-			color.Yellow.Fprintln(out, "Press Ctrl+C to exit")
-		}
+	if err := forwarderManager.Start(ctx); err != nil {
```

Comment:
should this under be `if r.runCtx.Opts.PortForward.Enabled {`  so we only start the forward manager if port forward is enabled?

Reply:
It's not needed. `r.createForwarder` creates an empty forwarder when `!r.runCtx.Opts.PortForward.Enabled`